packages/mwp-core/src/renderers/server-render.jsx

```diff
@@ -53,6 +54,34 @@ const resolveSideEffects = () => ({
 	notFound: NotFound.rewind(),
 });
 
+/**
+ * Using the User agent string + an external library called Mobile-Device we detect if the user
+ * is attempting to render the page on the server using a mobile, tablet, or desktop device
+ * @param  {String} userAgent User agent string detected from headers
+ * @return {Object} Object of which category the devices falls within
```

Comment:
you can drop the JSDoc tags now that you have Flowtypes - don't want to have to keep track of redundant type annotations